setup.py

```diff
@@ -47,7 +47,7 @@
           activecollab=["pypandoc", "pyac>=0.1.5"],
           bts=["PySimpleSOAP","python-debianbts>=2.6.1"],
           trac=["offtrac"],
-          bugzilla=["python-bugzilla"],
+          bugzilla=["python-bugzilla>=2.1.0"],
```

Comment:
Do we have to add this constraint? Couldn't one continue using password authentication with older versions?